packages/data-stores/src/reader/types/index.ts

```diff
@@ -131,3 +131,23 @@ export type PendingPostSubscriptionsResult = {
 	pendingPosts: PendingPostSubscription[];
 	totalCount: number;
 };
+
+export type SiteSubscriptionDetails = {
+	ID: string;
+	blog_ID: string;
+	name: string;
+	URL: string;
+	site_icon: string;
+	date_subscribed: Date;
+	delivery_methods: SiteSubscriptionDeliveryMethods;
+};
+
+export type SiteSubscriptionDetailsAPIResponse = {
+	ID: string;
+	blog_ID: string;
+	name: string;
+	URL: string;
+	site_icon: string;
+	date_subscribed: Date;
+	delivery_methods: SiteSubscriptionDeliveryMethods;
+};
```

Comment:
For now, both types look the same, this may change later as this is just first quick iteration.